skaffold.yaml

```diff
@@ -1,63 +1,71 @@
-apiVersion: skaffold/v2beta24
+apiVersion: skaffold/v4beta2
 kind: Config
 
 requires:
   - path: examples/secrets/skaffold.yaml
+  - path: examples/values/serviceAccounts/skaffold.yaml
 
 build:
   artifacts:
     - image: substra/substra-backend
       context: .
-      docker:
-        dockerfile: docker/substra-backend/Dockerfile
       sync:
         manual:
         - src: backend/**/*
           dest: /usr/src/app
           strip: backend/
+      docker:
+        dockerfile: docker/substra-backend/Dockerfile
 
-deploy:
-  statusCheckDeadlineSeconds: 600
+manifests:
   helm:
     releases:
       - name: backend-org-1
         chartPath: charts/substra-backend
+        valuesFiles:
+        - examples/values/backend-org-1.yaml
         namespace: org-1
+        setValues:
+          _: _
+        setValueTemplates: &image-tags
+          api.events.image: &image-params
+            registry: '{{.IMAGE_DOMAIN_substra_substra_backend}}'
+            repository: '{{.IMAGE_REPO_NO_DOMAIN_substra_substra_backend}}'
+            tag: '{{.IMAGE_TAG_substra_substra_backend}}'
+          scheduler.image: *image-params
+          schedulerWorker.image: *image-params
+          server.image: *image-params
+          worker.events.image: *image-params
+          worker.image: *image-params
         createNamespace: true
-        valuesFiles: [ examples/values/backend-org-1.yaml ]
-        setValues: { _: _ } # to support profiles
-        imageStrategy:
-          helm:
-            explicitRegistry: true
-        artifactOverrides:
-          server.image: substra/substra-backend
-          scheduler.image: substra/substra-backend
-          worker.image: substra/substra-backend
-          schedulerWorker.image: substra/substra-backend
-          worker.events.image: substra/substra-backend
-          api.events.image: substra/substra-backend
-
+        skipBuildDependencies: true
```

Comment:
So why do we use `skipBuildDependencies` and instead have a hook for that?

Reply:
Because it calls `helm dep update` and not `helm dep build`